workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -79,3 +80,75 @@ If the `delegateIp` option is used, then - if available - an IP address assigned
 |Options||
 |--|--|
 | `delegateIp` | `pod` backend only: The IP assigned to the pod will be delegated to the VM using DHCP |
+
+### `slirp` connection
+
+This connection will use the qemu user interface.
+Let the VM be exposed like a process which would be running inside a container. The requirement is that a process will bind to a port.
+
+It is a known issue that SLIRP is not recommended for anything in production.
```

Comment:
for anything in production -> for production

Reply:
Or if we strive to shortness and clarity then maybe replace the whole sentence with: "Note: SLIRP is not recommended for production use."